app/components/LicensePage.tsx

```diff
@@ -205,8 +235,10 @@ export default class LicensePage extends Component {
         this.setState({ error: null });
     }
 
-    onLicenseEdit(proxy, field) {
-        this.setState(Form.fieldNameValue(field));
+    onLicenseEdit(_proxy: any, field: Field) {
```

Comment:
I guess we can use here cool TS feature to get type from a function argument:
```suggestion
    onLicenseEdit(_proxy: any, field: Parameters<typeof Stage.Basic.Form.fieldNameValue>[0]) {
```
and avoid defining `Field` type.

Ref.: https://stackoverflow.com/questions/51851677/how-to-get-argument-types-from-function-in-typescript

Reply:
applied adjustment: https://github.com/cloudify-cosmo/cloudify-stage/pull/2012/commits/3f8a17a693b9233c677d5c097e00bca99f5e93a5